app/helpers/htmlFormHelpers.php

```diff
@@ -76,7 +78,7 @@ function caHTMLSelect($ps_name, $pa_content, $pa_attributes=null, $pa_options=nu
 		
 		$vs_element = "<select name='{$ps_name}' {$vs_attr_string}>\n";
 		
-		$vs_selected_val = isset($pa_options['value']) ? $pa_options['value'] : null;
```

Comment:
This is there to allow a single value to be passed via the "value" option. Multiple values can be passed with the "values" option.